learn_script.py

```diff
@@ -31,8 +48,7 @@ def prompt():
     ansiprint(
         "<bold><green>[In]</green></bold> <light-blue>"
         + os.path.abspath(".")
-        + "</light-blue>$ ",
-        end="",
+        + "</light-blue>$ "
```

Comment:
keep the end of line empty. we want to simulate a shell here. otherwise it will skip a line.

```suggestion
        + "</light-blue>$ ",
        end="",
```